plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -239,6 +238,10 @@ public Stream<AddFileEntry> getActiveFiles(ConnectorSession session, TableSnapsh
         return retrieveActiveFiles(session, tableSnapshot, metadataEntry, protocolEntry, TupleDomain.all(), alwaysTrue());
     }
 
+    /**
+     * Retrieves the active file entries matching the specified partition constraint
+     * along with the column statistics for the specified projected columns only.
```

Comment:
javadoc should also say what `projectedColumns = Optional.empty` means.

(and per offline discussion, it seems that we don't need Optional at all, if Optional.empty means the same thing as Optiona.of(emptySet)